draft-ietf-quic-transport.md

```diff
@@ -1355,6 +1355,9 @@ long enough to fulfill the minimum size for every QUIC version it supports.
 The client populates the Source Connection ID field with a value of its choosing
 and sets the SCIL field to match.
 
+The same Destination and Source Connection ID values are used for any 0-RTT
+packets the client sends.
+
```

Comment:
This could be read as if the ID is the same in both directions.

Reply:
Agreed, this is not completely clear.  How about: "The first flight of 0-RTT packets use the same Destination and Source Connection ID values as the client's first Initial."?

Also, on 1356, "sets the SCIL field to match." is not very clear.  I'd either drop the clause or say "and sets the SCIL field to indicate the length."